src/css/my-widgets.scss

```diff
@@ -389,28 +389,55 @@
 		opacity: 0.5;
 	}
 
-	.page .controls div.delete_dialogue {
-		background: url(../../../img/delete-dialogue-bg.png) no-repeat center;
-		padding: 20px 0;
-		margin-bottom: 5px;
+	.page .controls .delete_dialogue {
```

Comment:
removed the background image and replaced w/ a pure css solution. I wanted a little more space in the dialog and after realizing it was an image, figured it was worth the effort to ditch the image here.